src/usr/local/www/services_captiveportal_filemanager.php

```diff
@@ -276,14 +276,14 @@ function cpelements_sort() {
 <div class="infoblock panel panel-default">
 	<div class="panel-heading"><h2 class="panel-title"><?=gettext("Notes");?></h2></div>
 	<div class="panel-body">
-	<?=gettext("Any files that you upload here with the filename prefix of captiveportal- will " .
+	<?=gettext("Any files that are uploaded here with the filename prefix of captiveportal- will " .
 	"be made available in the root directory of the captive portal HTTP(S) server. " .
-	"You may reference them directly from your portal page HTML code using relative paths. " .
-	"Example: you've uploaded an image with the name 'captiveportal-test.jpg' using the " .
-	"file manager. Then you can include it in your portal page like this:")?><br /><br />
+	"They may be referenced directly from the portal page HTML code using relative paths. " .
+	"Example: An image uploaded with the name 'captiveportal-test.jpg' using the " .
+	"file manager. Can then be included in the portal page like this:")?><br /><br />
```

Comment:
You can remove the full stop and have a lowercase "c"
"file manger can then be included in the portal page like this:"